### ISSUE TITLE
MRI-ESM2-0-tasmax.yaml fails cleaning step for ssp126 and ssp585 'cause "AssertionError: there are nans!"

### ISSUE THREAD

**Issue by brews** [ClimateImpactLab/downscaleCMIP6#370]
MRI-ESM2-0 tasmax ssp126 and ssp585 are failing new cmip6-cleaning validation tests. "There are nans!"

Archived workflow is here: https://argo.cildc6.org/archived-workflows/default/aebb4a2b-4e8d-4285-ba42-6b6b979be1d2.

Logged error for ssp126:
```
Validating gs://clean-b1dbca25/cmip6/ScenarioMIP/MRI/MRI-ESM2-0/ssp126/r1i1p1f1/day/tasmax/gn/v20191108.zarr
/opt/dodola/dodola/core.py:652: UserWarning: projection cmip6 file has excess timesteps, has 39055
  warnings.warn(
Traceback (most recent call last):
  File "/argo/staging/script", line 44, in <module>
    tasks = dask.compute(*tasks)
  File "/opt/conda/lib/python3.9/site-packages/dask/base.py", line 570, in compute
    results = schedule(dsk, keys, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/dask/threaded.py", line 79, in get
    results = get_async(
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 507, in get_async
    raise_exception(exc, tb)
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 315, in reraise
    raise exc
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 220, in execute_task
    result = _execute_task(task, data)
  File "/opt/conda/lib/python3.9/site-packages/dask/core.py", line 119, in _execute_task
    return func(*(_execute_task(a, cache) for a in args))
  File "/argo/staging/script", line 26, in clear_memory_intensive_tests
    _test_for_nans(d, v)
  File "/opt/dodola/dodola/core.py", line 634, in _test_for_nans
    assert ds[var].isnull().sum() == 0, "there are nans!"
AssertionError: there are nans!
```

Logged error for ssp585:

```
Validating gs://clean-b1dbca25/cmip6/ScenarioMIP/MRI/MRI-ESM2-0/ssp585/r1i1p1f1/day/tasmax/gn/v20191108.zarr
/opt/dodola/dodola/core.py:652: UserWarning: projection cmip6 file has excess timesteps, has 39055
  warnings.warn(
Traceback (most recent call last):
  File "/argo/staging/script", line 44, in <module>
    tasks = dask.compute(*tasks)
  File "/opt/conda/lib/python3.9/site-packages/dask/base.py", line 570, in compute
    results = schedule(dsk, keys, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/dask/threaded.py", line 79, in get
    results = get_async(
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 507, in get_async
    raise_exception(exc, tb)
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 315, in reraise
    raise exc
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 220, in execute_task
    result = _execute_task(task, data)
  File "/opt/conda/lib/python3.9/site-packages/dask/core.py", line 119, in _execute_task
    return func(*(_execute_task(a, cache) for a in args))
  File "/argo/staging/script", line 26, in clear_memory_intensive_tests
    _test_for_nans(d, v)
  File "/opt/dodola/dodola/core.py", line 634, in _test_for_nans
    assert ds[var].isnull().sum() == 0, "there are nans!"
AssertionError: there are nans!
```

Blocks progress on run #206.

**Comment by brews** [comment]
Heads up to @dgergel.

**Comment by dgergel** [comment]
I looked into this and ssps 126 and 585 have NaNs for all gridcells for the entire future projection period. Recommend eliminating those ssps for MRI-ESM2-0, and I've created two related issues: 1) updating the parameter files, 2) finding out if there is an updated version in Pangeo and if not, tracking down why these SSPs entirely have NaNs with ESGF and/or CSIRO-ARCCSS. We can confine our first round deliverable for this model to SSPs 245 and 370.

Updated and closed with #375 and #376
